lib/msal-browser/test/cache/BrowserCacheManager.spec.ts

```diff
@@ -3417,641 +3412,5 @@ describe("BrowserCacheManager tests", () => {
                 refreshToken: ["refreshToken2"],
             });
         });
-
-        describe("getAccountInfoByFilter", () => {
```

Comment:
This and getAccountInfoByHints APIs were removed and their logic is already covered in the tests for getAccountX APIs